### PR TITLE
Raise .env version to 5.0.3-14 to include ES 7.16.2

### PR DESCRIPTION
**Pull request by MrGeneration**
This raises the `.env` version to `5.0.3-14` which includes Elasticsearch version 7.16.2 to complete the security related raise by https://github.com/zammad/zammad-docker-compose/pull/265

### CODE REVIEW

**Comment by MrGeneration** [comment]
After this PR is merged I can handle this PR https://github.com/zammad/zammad-docker-compose/pull/267 by cherry picking the changes of this PR. Other wise tests will keep failing.

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ To run Zammad behind a revers proxy, we provide `docker-compose.proxy-example.ym
 
 See `.examples/proxy/docker-compose.yml` for an example proxy project.
 
-Like this, you can add your `docker-compose.prod.yml` to a branch of your Git repository and stay up to date by merging changes to your branch.
+Like this, you can add your `docker-compose.prod.yml` to a branch of your Git repositorysitory and stay up to date by merging changes to your branch.
```
Comment (monotek) [inline README.md:33]:
```suggestion
Like this, you can add your `docker-compose.prod.yml` to a branch of your Git repository and stay up to date by merging changes to your branch.
```

File: README.md [hunk 2]
```diff
@@ -2,17 +2,17 @@
...
 Zammad is a web based open source helpdesk/ticket system with many features
 to manage customer communication via several channels like telephone, facebook,
-twitter, chat and e-mails. It is distributed under the GNU AFFERO General Public
- License (AGPL). Do you receive many e-mails and want to answer them with a team of agents?
+twitter, chat and emails. It is distributed under the GNU AFFERO General Public
+ License (AGPL). Do you receive many emails and want to answer them with a team of agents?
 You're going to love Zammad!
 
-## Use case for this repo
+## Use case for this repositorysitory
```
Comment (monotek) [inline README.md:9]:
```suggestion
## Use case for this repository
```

**Review by monotek — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (MrGeneration) [reply README.md:9]: What the.... what happened here :(

**Comment by MrGeneration** [comment]
I am so sorry, no idea what happened there. Fixed!

**Review by monotek — APPROVED** [review]

**Pull request merged**